minecraft.js

```diff
@@ -66,6 +66,25 @@ app.post('/command', (req, res) => {
   }, 250);
 });
 
+app.get('/properties', async (req, res) => {
+  try {
+    const result = await getProperties('./server.properties');
+    res.send(result);
+  } catch (e) {
+    res.status(500).send();
+  }
+});
+
+app.put('/properties', async (req, res) => {
+  try {
+    const properties = req.body;
+    writeProperties('./server.properties', properties);
```

Comment:
Make sure you `await` this

Reply:
Ah yes, thank you for catching that!! I've made that change.